src/bbmap_bbsplit/config.vsh.yaml

```diff
@@ -1,4 +1,4 @@
-namespace: "bbmap"
+# namespace: "bbmap"
```

Comment:
Will more bbmap components be added? If so, it would be better to move bbmap to a different namespace and rename the component.

Otherwise, this change is ok.